cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandlerInternal.java

```diff
@@ -200,4 +210,41 @@ public void getAppDetailForVersion(HttpRequest request, HttpResponder responder,
         : applicationLifecycleService.getAppDetail(appId);
     responder.sendJson(HttpResponseStatus.OK, GSON.toJson(appDetail));
   }
+
+  /**
+   * Update git metadata of the provided applications.
+   *
+   * @param request {@link FullHttpRequest}
+   * @param responder {@link HttpResponse}
+   * @param namespace the namespace where the applications are deployed
+   * @throws Exception if either namespace or any of the application doesn't exist
+   */
+  @POST
+  @Path("/apps/updateGitMetadata")
```

Comment:
in ApplicationDetail, it is called 'sourceControlMeta'. Should update this to be consistent, updateSourceControlMeta

Reply:
done